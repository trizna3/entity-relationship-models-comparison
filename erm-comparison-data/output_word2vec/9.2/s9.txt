weather-EMPTY
games-game
tile type-tile type
units-unit
fuel type-fuel type
boards-board
tile-tiles
