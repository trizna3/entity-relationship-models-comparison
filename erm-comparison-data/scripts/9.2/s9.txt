entity set game {
};

entity set board {
	"rows"
	"columns"
	"move count"
};

entity set tiles {
};

entity set "tile type" {
	"type"
};

entity set unit {
	"name"
	"health"
	"attack"
	"defense"
	"action points"
};

entity set "fuel type" {
	"type"
};

entity set difficulty {
	"action points needed"
};

association between 
{
	game ONE
	board MANY
}{};

association between 
{
	board ONE
	tiles MANY
}{};

association between 
{
	tiles MANY
	"tile type" MANY
}{};

association between 
{
	"tile type" MANY
	board ONE
}{};

association between 
{
	"tile type" MANY
	unit ONE
}{};

association between 
{
	unit MANY
	"fuel type" ONE
}{};

association between 
{
	"tile type" ONE
	difficulty MANY
}{};
