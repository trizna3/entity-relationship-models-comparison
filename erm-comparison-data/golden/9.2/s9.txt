games-game
boards-board
tile-tiles
tile type-tile type
weather-EMPTY
fuel type-fuel type
units-unit